api/explored/server.go

```diff
@@ -114,5 +196,11 @@ func NewServer(cm ChainManager, s Syncer, tp TransactionPool) http.Handler {
 
 	mux.GET("/api/consensus/tip", srv.consensusTipHandler)
 
+	mux.GET("/api/explorer/element/siacoin", srv.explorerElementSiacoinHandler)
```

Comment:
I generally prefer `/siacoin/:id` over query params for endpoints like this. Do you have a preference @lukechampine?

Reply:
yeah, any sort of stable ID (element ID, txid, block ID) should be part of the URL